packages/sdk/src/sdk/sdk.ts

```diff
@@ -144,6 +143,8 @@ const initializeServices = (config: SdkConfig) => {
   })
   const logger = config.services?.logger ?? defaultLogger
 
+  const isBrowser: boolean =
```

Comment:
`browser-or-node` dep was literally just doing this. not worth carrying a whole other package for one usage.